exercises/secret-handshake/description.md

```diff
@@ -23,7 +23,4 @@ Here's a couple of examples:
 Given the input 3, the function would return the array
 ["wink", "double blink"] because 3 is 11 in binary.
 
-Given the input 19, the function would return the array
-["double blink", "wink"] because 19 is 10011 in binary.
-Notice that the addition of 16 (10000 in binary)
-has caused the array to be reversed.
+The addition of 16 (10000 in binary) would reverse the array. So given the input 19, the function would return the array ["double blink", "wink"] because 19 is 10011 in binary.
```

Comment:
```suggestion
The addition of 16 (`10000` in binary) would reverse the array. So given the input 19, the function would return the array ["double blink", "wink"] because 19 is `10011` in binary.
```